rebuild/lib/index.js

```diff
@@ -5,6 +5,36 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 const bindings = require("bindings")("blst_ts_addon");
 
+bindings.verify = function verify(msg, pk, sig) {
+  return bindings.aggregateVerify([msg], [pk], sig);
+};
+
+bindings.asyncVerify = function asyncVerify(msg, pk, sig) {
+  return bindings.asyncAggregateVerify([msg], [pk], sig);
+};
+
+bindings.fastAggregateVerify = function fastAggregateVerify(msg, pks, sig) {
+  let key;
+  try {
+    // this throws for invalid key, catch and return false
+    key = bindings.aggregatePublicKeys(pks);
+  } catch {
+    return false;
+  }
+  return bindings.aggregateVerify([msg], [key], sig);
```

Comment:
can we move this L24 inside try catch itself?

Reply:
I think these two should throw so they are the same behavior as the other verifies?  Or perhaps should all of them not throw?  I will turn on the spec tests in the next PR to make sure we are compliant.  I forget all of the cases off the top of my head but I think none of them should throw on first blush